File: 프로그래머스/1/17681. ［1차］ 비밀지도/［1차］ 비밀지도.py
def solution(n, arr1, arr2):
    answer = []
    
    li = [a | b for a,b in zip(arr1, arr2)]
    
    for i in li:
        s = str(bin(i))
        s = s[2:].zfill(n)
        s = s.replace('1', '#')
        s = s.replace('0', ' ')
        answer.append(s)
        
    return answer

File: 프로그래머스/1/17681. ［1차］ 비밀지도/［1차］ 비밀지도.cpp
#include <string>
#include <vector>
#include <iostream>

using namespace std;

vector<string> solution(int n, vector<int> arr1, vector<int> arr2) {
    vector<string> answer;
    
    vector<int> arr;
    for (int i = 0; i < n; i++) {
        arr.push_back(arr1[i] | arr2[i]);
    }
    
    for (int i = 0; i < n; i++) {
        string s = "";
        for (int j = 0; j < n; j++) {
            if (arr[i] & (1 << j)) {
                s = "#" + s;
            } else {
                s = " " + s;
            }
        }
        answer.push_back(s);
    }
    
    return answer;
}